### ISSUE TITLE
[BUG] Web UI

### ISSUE THREAD

**Issue by valsha** [bunkerity/bunkerweb#269]
![image](https://user-images.githubusercontent.com/7966513/177045934-370217fa-eb1a-455a-9a0d-104c72057ae0.png)
Web UI Linux integration - Not Found 404

**variables.env**
```
HTTP_PORT=80
HTTPS_PORT=443
DNS_RESOLVERS=8.8.8.8 8.8.4.4

SERVER_NAME=bwadm.mysite.com
MULTISITE=yes
USE_API=yes
API_WHITELIST_IP=127.0.0.0/8
bwadm.mysite.com_USE_UI=yes
bwadm.mysite.com_USE_REVERSE_PROXY=yes
bwadm.mysite.com_REVERSE_PROXY_URL=/botcheck
bwadm.mysite.com_REVERSE_PROXY_HOST=http://myui:7000
bwadm.mysite.com_REVERSE_PROXY_HEADER=X-Script-Name /botcheck
bwadm.mysite.com_REVERSE_PROXY_INTERCEPT_ERRORS=no
```

**bunkerweb-ui.env**
```
ADMIN_USERNAME=user
ADMIN_PASSWORD=mypasswd
ABSOLUTE_URI=http://bwadm.mysite.com/botcheck/

```

**Comment by fl0ppy-d1sk** [comment]
Hello @valsha,

There is an error in the documentation, can you try to replace **http://myui:7000** with **http://127.0.0.1:7000** ?

**Comment by fl0ppy-d1sk** [comment]
Also @valsha, which OS are you using ?

**Comment by TheophileDiot** [comment]
> 

✅ It has been fixed

**Comment by TheophileDiot** [comment]
> Hello @valsha,
> 
> There is an error in the documentation, can you try to replace **http://myui:7000** with **http://127.0.0.1:7000** ?

✅ It has been fixed

**Comment by valsha** [comment]
> Also @valsha, which OS are you using ?

Ubuntu 22.04

**Comment by fl0ppy-d1sk** [comment]
Hello @valsha, it should be fixed with the last v1.4.3 release.
